avogadro/qtplugins/gamessinput/gamessinputdialog.cpp

```diff
@@ -376,7 +456,7 @@ void GamessInputDialog::setBasicDefaults()
   ui.titleEdit->setText(QString());
   ui.calculateCombo->setCurrentIndex(CalculateSinglePoint);
   ui.theoryCombo->setCurrentIndex(TheoryB3LYP);
```

Comment:
Should we set a different default? IMHO there are some better functionals now...

Reply:
For example, I'd probably suggest B97-D or wB97X-D